x-pack/plugins/siem/public/components/tables/helpers.tsx

```diff
@@ -87,7 +87,12 @@ export const getRowItemDraggables = ({
       const id = escapeDataProviderId(`${idPrefix}-${attrName}-${rowItem}`);
       return (
         <React.Fragment key={id}>
-          {index !== 0 ? <>,&nbsp;</> : null}
+          {index !== 0 ? (
+            <>
+              {', '}
+              <Spacer />
```

Comment:
I should be able to clean this up by removing the ternary and using just a `&&` and remove the extra space after the `,` since the spacer is here.